### DISCUSSION TITLE
1.4.2rc1

### DISCUSSION THREAD

**Discussion by villebro** [apache/superset#19298] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 1.4.2. This release
candidate features the following cherries on top of 1.4.1:

- #19248: feat: add support for comments in adhoc clauses (@villebro)
- #18214: feat(docker-compose): add TAG option (@villebro)
- #17641: fix(sqla): make text clause escaping optional (@villebro)
- #18566: fix(plugin-chart-echarts): area chart opacity bug (@villebro)

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/1.4.2rc1/

Git tag for the release:
https://github.com/apache/superset/tree/1.4.2rc1

The Change Log for the release:
https://github.com/apache/superset/blob/1.4.2rc1/CHANGELOG.md

The Updating instructions for the release (unchanged since 1.4.1):
https://github.com/apache/superset/blob/1.4.2rc1/UPDATING.md

public keys are available at:

https://www.apache.org/dist/superset/KEYS

The vote will be open for at least 72 hours or until the necessary number
of votes are reached.

Please vote accordingly:

[ ] +1 approve
[ ] +0 no opinion
[ ] -1 disapprove with the reason

Thanks,
The Apache Superset Team

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/1.4.2rc1'>1.4.2rc1</a>.</em>

**Comment by sujiplr** [comment]
[ ] +1 approve
